kustomize/commands/edit/remove/removeconfigmap.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package remove
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/konfig"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v4/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+type removeConfigMapOptions struct {
+	configMapNamesToRemove []string
+}
+
+// newCmdRemoveResource remove the name of a file containing a resource to the kustomization file.
+func newCmdRemoveConfigMap(fSys filesys.FileSystem) *cobra.Command {
+	var o removeConfigMapOptions
+
+	cmd := &cobra.Command{
+		Use: "configmap",
+		Short: "Removes specified configmap" +
+			konfig.DefaultKustomizationFileName(),
```

Comment:
This should say `Removes specified configmap from kustomization.yaml`

It currently reads as `Removes specified configmapkustomization.yaml`